SecurityAdvisoriesBuilder-hs.cabal

```diff
@@ -0,0 +1,60 @@
+-- This file has been generated from package.yaml by hpack version 0.28.2.
```

Comment:
Given that `hpack` is used to generate this, should this be under `.gitignore`?

Reply:
Yes. Last time I bootstrapped a package with stack it actually created a .gitignore with the cabal file excluded. O.o